components/response-filter/src/main/java/com/wso2telco/dep/responsefilter/ResponseFilterMediator.java

```diff
@@ -73,20 +76,22 @@ public String findFilterSchema(String sp, String application, String api, String
                 throw new Exception("database connection error");
```

Comment:
Return SQLException instead of Exception here

Reply:
done.